.codeclimate.yml

```diff
@@ -0,0 +1,162 @@
+# This is our default .CodeClimate.yml, broken out by language. Uncomment the
+# sections at the bottom that apply to your project. ACTION comments indicate
+# places where config might need to be tweaked.
```

Comment:
It would be nice to have a link to the "master" file this is taken from

Reply:
I can do that - I'll be going through with tweaks (turning on bundler-audit or brakeman for example, maybe tweaking rules). I'll add the link to the master file as I do those things. (It's https://github.com/cyberark/conjur-template/blob/master/.codeclimate.yml if you need it sooner.)